pom.xml

```diff
@@ -317,15 +317,10 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <version>${surefire.version}</version>
-            </plugin>
```

Comment:
Why was this removed?

Reply:
We already define surefire-plugin in profiles (default & contract-tests) and this is redundant